### PR TITLE
[TEST] Remove install_requires dependency with future

### PR DESCRIPTION
**Pull request by jantonguirao**
Signed-off-by: Joaquin Anton <janton@nvidia.com>

#### Why we need this PR?
*Pick one*
- It fixes a bug *bug description*
- It adds new feature needed because *why we need this feature*
- Refactoring to improve *what*

#### What happened in this PR?
 - Explain solution of the problem, new feature added.
 - What was changed, added, removed?
 - What is most important part that reviewers should focus on?
 - Was this PR tested? How?
 - Were docs and examples updated, if necessary?

**JIRA TASK**: [DALI-XXXX]

### CODE REVIEW

**Comment by jantonguirao** [comment]
!build

File: dali/python/setup.py.in [hunk 1]
```diff
@@ -33,9 +33,5 @@ setup(name='nvidia-dali@DALI_FLAVOR_MINUS@',
           'console_scripts': [
               'rec2idx = rec2idx:main',
               ],
-          },
-      install_requires = [
-          'future',
```
Comment (JanuszL) [inline dali/python/setup.py.in]: It may fail for python 2.7 (like new print or division).

Comment (jantonguirao) [reply dali/python/setup.py.in]: I am trying to figure out whether this is really necessary. This suggests `future` is required for the installation script itself.

**Comment by jantonguirao** [comment]
future is required, test completed

**Pull request closed without merging**